frontend/src/metabase/containers/Form.jsx

```diff
@@ -215,18 +219,16 @@ export default class Form extends React.Component {
   };
 
   componentDidUpdate(prevProps: Props, prevState: State) {
-    if (!this.props.form) {
```

Comment:
@tlrobinson I removed this line, but I don't understand what it was doing in the first place. Any idea what might have broken that I should check?